lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -41,9 +41,9 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
       description: 'Properly size images',
       informative: true,
       helpText:
-        'Image sizes served should be based on the device display size to save network bytes. ' +
-        'Learn more about [responsive images](https://developers.google.com/web/fundamentals/design-and-ui/media/images) ' +
-        'and [client hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints).',
+        'Serve images that are smaller than the user\'s viewport to save cellular data ' +
```

Comment:
this may be nitpicky but since we're improving the description here... :) it's not just the viewport size but anywhere that's bigger than its rendered size, what about just "Serve images that are appropriately sized to save cellular data..." and all the nitty details can be at the Learn More?